src/multicloud-connector/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+1.0.0
+++++++
+* GA release
+* Register all required RPs before running each command
+* Reflect API swagger changes
+* Store the output of ``generate-aws-template`` to json file on disk
+* Update help guide with more examples
```

Comment:
Since the aaz autogenerated cmd's examples are changed, please make a pr to aaz repo to update the related aaz models here: https://github.com/Azure/aaz/tree/main/Commands/arc-multicloud.

If the model is already updated, please link the aaz repo update pr here.

Reply:
FYI - https://github.com/Azure/aaz/pull/577